python/hail/tests/tests.py

```diff
@@ -268,6 +268,8 @@ def test_dataset(self):
                 (dataset.repartition(16, shuffle=False)
                  .same(dataset)))
 
+            self.assertTrue(dataset.naive_coalesce(2).same(dataset))
```

Comment:
`dataset` iterates through VDS and GDS. You only defined it on VDS - that's the source of the test failure now. It should probably be on GDS too, right?